reframe/utility/sanity.py

```diff
@@ -466,7 +466,7 @@ def assert_bounded(val, lower=None, upper=None, msg=None):
 
 
 @deferrable
-def assert_reference(val, ref, lower_thres=None, upper_thres=None, msg=None):
+def assert_reference(val, ref, lower_thres=None, upper_thres=None, tag=None):
```

Comment:
I see two problems here:

1. The `msg` is a valid parameter used in every sanity functions and it is an omission that it is ignored here. I will open an issue for that.
2. I don't like the idea about tag as an argument. What does it represent in the context of `assert_reference()`?

Reply:
A solution would be to implement the logic for the `msg` as for other sanity functions and have the caller, i.e., the `check_performance()` set the message using the tag. But again I'm not 100% sure.